infrastructure/aws/ecs.tf

```diff
@@ -25,8 +25,11 @@ locals {
     "DJANGO_SETTINGS_MODULE" : "redbox_app.settings",
     "DEBUG" : true,
     "AWS_REGION" : var.region,
-    "OPENAI_API_KEY" : var.openai_api_key,
     "FROM_EMAIL" : var.from_email,
+    "LLM__TYPE": var.llm_type,
+    "LLM__NAME": var.llm_name,
+    "LLM__API_BASE": var.llm_api_base,
+    "AZURE_API_KEY": var.azure_api_key
```

Comment:
Please change `AZURE_API_KEY` to `AZURE_OPENAI_API_KEY` and `LLM__API_BASE` to `AZURE_OPENAI_ENDPOINT` to match convention from Azure documentation: https://learn.microsoft.com/en-us/azure/ai-services/openai/gpt-v-quickstart?tabs=image%2Ccommand-line&pivots=programming-language-python#environment-variables

May seem nitpicky but some libraries look for this things in env (for example some eval libraries in notebooks)

Reply:
sounds reasonable @lmwilkigov

Reply:
[69ef416](https://github.com/i-dot-ai/redbox-copilot/pull/472/commits/69ef4168e8860e8bce385cb0cd7bd35416a2a43c)
[2cab6ba](https://github.com/i-dot-ai/redbox-copilot/pull/472/commits/2cab6baf941d61f885d6ea3e37caedc3d1588db3)